src/positions/types.ts

```diff
@@ -9,15 +9,28 @@ export interface PositionDisplayProps {
   imageUrl: string
 }
 
+export type DataProps = EarnDataProps
+
+export interface EarnDataProps {
+  apy: number
```

Comment:
I think not every pool will have an apy, some may just use apr. Should we make this optional so this can be compatible with new pools we had that don't have apy? cc @finnian0826 @jh2oman

Reply:
Do we want to display APR or APY depending on the pool?
Instead would it be possible to land on one so we are displaying comparable values across our earn pools?

Reply:
I believe @finnian0826 has been working with product on this, and from the latest designs presented this week, I think we wanted to include a total yield and a breakdown, although I can't find the link to the designs right now. @finnian0826 do you have it?

Reply:
yeah the designs for the breakdown are [here](https://www.figma.com/design/E1rC3MG74qEg5V4tvbeUnU/Stablecoin-Enablement?node-id=4761-18541&t=jjFc6ucvG3KMaiKG-0), I think we could include a compounding APY field, and a reward APR field, that should cover all of the different pool types I think.